tests/test_msg_thorchain_signtx.py

```diff
@@ -93,6 +93,53 @@ def test_sign_eth_btc_swap(self):
         self.assertEqual(hexlify(sig_r), '30c4d80ebf1c9e214370bae286a9cee9384788080bcfb773afb75cb2573ef433')
         self.assertEqual(hexlify(sig_s), '104f5b60cf6f8ab3a511fb98ddbbfdc9b8837a7b6d8d8627ca65510ca3733b08')
 
+
+    def test_sign_btc_add_liquidity(self):
+        self.requires_firmware("7.0.2")
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        inp1 = proto_types.TxInputType(address_n=[0],  # 14LmW5k4ssUrtbAB4255zdqv3b4w1TuX9e
+                             # amount=390000,
+                             prev_hash=unhexlify('d5f65ee80147b4bcc70b75e4bbf2d7382021b871bd8867ef8fa525ef50864882'),
+                             prev_index=0,
+                             )
+
+        out1 = proto_types.TxOutputType(op_return_data=b'ADD:BTC.BTC:thorpub1addwnpepq2ynqt500fag3wyxsjuv7570qxr8rqtpx93hw3cpqaqxtwxesy76utgtemp:420',
+                              amount=0,
+                              script_type=proto_types.PAYTOOPRETURN,
+                              )
+
+
+        (signatures, serialized_tx) = self.client.sign_tx('Bitcoin', [inp1, ], [out1, ])
```

Comment:
I think this is missing the actual asgard-spendable output... won't this just throw all your coins at the miner?